### ISSUE TITLE
Insufficient contrast in DatePicker for days adjacent to current month

### ISSUE THREAD

**Issue by rstrangfeld** [carbon-design-system/carbon#2504]
In the DatePicker, when displaying a month, the ending days of the previous month and beginning days of the next month are shown in light gray at the top and bottom of the calendar page. They are still clickable like the days in the current month.

DAP reports contrast violations for these light gray days, with the message:
"Minimum contrast of 2.11 with its background is not sufficient to meet WCAG AA requirements for text of size 12px and weight of 400."

The relevant WCAG criterion is Success Criterion 1.4.3 Contrast (Minimum).

This was found using carbon-components-react 6.71.0 and carbon-components 9.62.3.
